.github/workflows/CI.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - R ${{ matrix.R }} - ${{ matrix.os }} 
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        version:
+          - '1.0'
+          - '1.1'
```

Comment:
```suggestion
```

1.0 and 1.1 are not compatible with a Project.toml in the ./test directory

Reply:
well that explains that